### PR TITLE
SUBMARINE-502. [WEB] Implement outline of experiment information page with Angular

### PR DESCRIPTION
**Pull request by jasoonn**
### What is this PR for?
Implement outline of experiment information page.

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-502?filter=allopenissues

### How should this be tested?
https://travis-ci.org/github/apache/submarine/builds/688737391

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by jasoonn** [comment]
@liuxunorg @kobe860219 @chiajoukuo @kevin85421 Can you help me review the pr? Thanks!

**Comment by liuxunorg** [comment]
@jasoonn Please rebase this commit.

**Comment by jasoonn** [comment]
@liuxunorg Thank for your suggestion. I have updated my pr.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { ExperimentService } from '../../../../services/experiment.service';
+import { ExperimentInfo } from '@submarine/interfaces/experiment-info';
+import { NzMessageService } from 'ng-zorro-antd';
+
+@Component({
+  selector: 'app-experiment-info',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.ts:27]:
```suggestion
  selector: 'submarine-experiment-info',
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/metrics/metrics.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-metrics',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/metrics/metrics.component.ts:23]:
```suggestion
  selector: 'submarine-metrics',
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/hyper-params/hyper-params.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-hyper-params',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/hyper-params/hyper-params.component.ts:23]:
```suggestion
  selector: 'submarine-hyper-params',
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/charts/charts.component.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-charts',
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/charts/charts.component.ts:23]:
```suggestion
  selector: 'submarine-charts',
```

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+      <li nz-menu-item (click)="currentState = 2">Metrics</li>
+      <li nz-menu-item (click)="currentState = 3">Outpus</li>
+    </ul>
+  </nz-sider>
+  <nz-layout>
+    <div [ngSwitch]="currentState">
+      <app-charts *ngSwitchCase="0"></app-charts>
+      <app-hyper-params *ngSwitchCase="1"></app-hyper-params>
+      <app-metrics *ngSwitchCase="2"></app-metrics>
+      <app-outputs *ngSwitchCase="3"></app-outputs>
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/experiment/experiment-info/experiment-info.component.html:96]:
```suggestion
      <submarine-charts *ngSwitchCase="0"></submarine-charts>
      <submarine-hyper-params *ngSwitchCase="1"></submarine-hyper-params>
      <submarine-metrics *ngSwitchCase="2"></submarine-metrics>
      <submarine-outputs *ngSwitchCase="3"></submarine-outputs>
```

**Comment by kobe860219** [comment]
@jasoonn Thanks for your contribution. It looks good to me :))

**Comment by chiajoukuo** [comment]
Thanks for your contribution. It looks good to me.

**Comment by jasoonn** [comment]
@pingsutw @chiajoukuo Thank you for your suggestions. I have updated my pr.

**Comment by pingsutw** [comment]
+1 Thanks @jasoonn

**Pull request closed without merging**